### ISSUE TITLE
M4.1 Start migration of OSA to IRIS cloud

### ISSUE THREAD

**Issue by tms-epcc** [wfau/ScienceArchives#364]

**Comment by tms-epcc** [comment]
05/MAY/23
- O/G for past month
- processes still migrating to Somerville at ACF 
- speak to MarkH

**Comment by tms-epcc** [comment]
11/AUG/23
- Staus of plan from [19/JUN/23 meeting](https://www.wiki.ed.ac.uk/pages/viewpage.action?pageId=578199767) is 
- @MikeeRead 
- [ ] look over old-style OSA interface
- [ ] check it is working,
- [ ] update release history etc - **in progress**
- @astrodb 
- [ ] will then do the DNS switch once happy operational and switch OSA onto old interface: **about to start**
- [ ] Get old style interface operating on Somerville then switch over on to that. : **on hold till new version of openStack is rolled out**
- @stvoutsin 
- - [ ] in parallel to above, will look at new TAP service : **[in progress](https://app.zenhub.com/workspaces/science-archives-project-management-6437c97cbb0b3e891a24bc8c/board?filterLogic=all&labels=tap&releases=Z2lkOi8vcmFwdG9yL1JlbGVhc2UvOTE1NjU&repos=274641225,zh-Z2lkOi8vcmFwdG9yL1JlcG9zaXRvcnkvMTMzNDE5NjE2)**

**Comment by tms-epcc** [comment]
25/AUG/23
- waiting on Mike to ok the switch
- also had to update the apache config

**Comment by tms-epcc** [comment]
08/SEP/23
- @astrodb still waiting on @MikeeRead but away Mike away until mid-Sept

**Comment by tms-epcc** [comment]
22/SEP/23
- release history done (see above)

**Comment by tms-epcc** [comment]
O6/OCT/23
- DNS switch done and old style interface now working on Somerville following the Openstack roll out 
- Next step need to move DBs over now once sufficient storage space available and SQL server license sorted

**Comment by tms-epcc** [comment]
17/APR/24
- #359 is for completion of the migration
